src/components/login/Login.js

```diff
@@ -52,7 +52,9 @@ class Login extends Component {
     }
 
     redirectCreateAccount = () => {
-        this.props.history.push('/create')
+        const { success_url } =  this.props.account.url
+        sessionStorage.setItem("success_url", success_url)
+        this.props.history.push(`/create`)
```

Comment:
these changes are unnecessary, it's been already done here:
https://github.com/near/near-wallet/blob/master/src/components/Routing.js#L113